src/containers/DeveloperPortal/containers/SdkAndLibraries/components/ConfirmationModal/ConfirmationModal.scss

```diff
@@ -0,0 +1,65 @@
+@import 'styles/font-mixins';
+
+.ConfirmationModal {
+  background-color: rgba(0, 0, 0, 0.3);
+  bottom: 0;
+  display: grid;
+  left: 0;
```

Comment:
can we fix the height of the modal? as there is a slight content shift when the modal is shown.